presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -701,6 +702,31 @@ public Node visitRevoke(SqlBaseParser.RevokeContext context)
                 context.grantee.getText());
     }
 
+    @Override
+    public Node visitShowGrant(SqlBaseParser.ShowGrantContext context)
+    {
+        Optional<ShowGrant.IdentityType> identityType = Optional.empty();
+        Optional<String> identity = Optional.empty();
+        Optional<QualifiedName> tableName = Optional.empty();
+
+        if (context.USER() != null) {
+            identityType = Optional.of(ShowGrant.IdentityType.USER);
+            identity = Optional.of(context.identifier().getText());
+        }
+
+        if (context.ALL() == null) {
+            tableName = Optional.of(getQualifiedName(context.qualifiedName()));
```

Comment:
The current implementation handles only table names and doesn't handle schema names. The current `grant/revoke` methods also don't handle schemas (and views). So we could handle schemas later?